src/main/resources/org/kuali/kra/award/AwardSpringBeans.xml

```diff
@@ -827,7 +767,7 @@
                 <bean class="org.kuali.kra.award.document.authorizer.MaintainAwardReportTrackingAuthorizer" parent="parentAwardAuthorizer">
                     <property name="taskName" value="maintainReportTracking" />
                 </bean>    
-                <bean class="org.kuali.kra.award.document.authorizer.CreateAwardAccountAuthorizer" parent="parentAwardAuthorizer">
+                <bean class="org.kuali.kra.award.document.authorizer.CreateAwardAccountAuthorizer" parent="parentAwardAuthorizer">mvn c

```

Comment:
What's this?  Looks like the beginning of a maven command at the end of this line?

Reply:
Ah thanks Julie! Good catch... didn't see that one